sections/battlefield.tex

```diff
@@ -19,29 +19,26 @@
 
   You can also use the Battlefield Board instead of the Combat Board in other game modes.
 
+  \subsection*{Victory Condition}
+  Victory in Adventure and Skirmish is achieved by defeating all enemy units in the final Combat.
+
   \begin{center}
-    {\transparent{0.2}{\includegraphics[width=0.8\linewidth]{\art/quiet_eye_of_the_dragon.png}}}
+    {\hfill\transparent{0.2}{\includegraphics[width=0.75\linewidth]{\art/land_mine.png}}}
   \end{center}\par
 
-  \vspace*{\fill}
   \columnbreak
 
-  \subsection*{Victory Condition}
-  Victory in Adventure and Skirmish is achieved by defeating all enemy units in the final Combat.
-
   \subsection*{Rule Ambiguity}
   \textbf{Note:} Both game modes were designed for the \textbf{Core Game only}.
-  As a result, the official rules do not address many interactions, particularly those involving expansion content.
-  If you wish to include expansion cards, you will need to establish house rules for unresolved interactions.
-
-  \smallskip
-  Such unanswered questions include:
+  As a result, the official rules do not address many interactions, particularly those involving expansion content. The game developers have \pagetarget{battlefield card clarifications}{clarified the following interactions}:
   \begin{itemize}
-    \item Which cards should be removed from the game apart from Diplomacy and Logistics?
-    \item Which cards should be removed in Skirmish Mode because they provide no Combat benefit?
-    \item How should cards with Permanent effects be handled in Adventure Mode?
-    \item How should cards with a ``Remove'' effect (e.~g.~Expert Learning) be handled in Adventure Mode?
-    \item How should cards with both Combat and non-Combat effects (e.~g.~Cape of Velocity) be handled in Adventure Mode?
+    \item The official rules remove only Diplomacy and Logistics ability cards, but common sense applies: the developers also recommend removing Ballistics and Pathfinding, as none provide benefit in Battlefield modes.
+    \item In Skirmish Mode, when a card with no Combat value is drawn, both players may mutually agree to discard it and draw a replacement.
+    \item All cards in the Adventure phase are single-use.
+      Consequently, resource-generating Permanent \svg[7]{permanent} cards (e.g.,~Inexhaustible Cart of Ore) are limited to their bottom \svg{instant} effect only.
```

Comment:
That's not how I read his answer.

For this specific card, the player is free to choose either 1 stone, or 3 stones, but has to remove the card either way.

For some other card, the choice can be more meaningful

Reply:
Yeah, for me "a good deal" sounds like a strategy advice, that's why. Not sure what other cases of permanent cards would look like with this